go-controller/pkg/ovn/egressgw_test.go

```diff
@@ -28,6 +28,24 @@ import (
 	"github.com/urfave/cli/v2"
 )
 
+func checkAPBRouteStatus(fakeOVN *FakeOVN, policyName string, expectFailure bool) {
```

Comment:
nit maybe this should go in external_gateway_apb_test.go since that is the apb specific test file